nav2_docking/opennav_docking/src/docking_server.cpp

```diff
@@ -40,6 +40,9 @@ DockingServer::DockingServer(const rclcpp::NodeOptions & options)
   declare_parameter("fixed_frame", "odom");
   declare_parameter("dock_backwards", false);
   declare_parameter("dock_prestaging_tolerance", 0.5);
+  declare_parameter("initial_rotation", true);
+  declare_parameter("initial_rotation_min_angle", 0.5);
+  declare_parameter("dock_backwards_without_sensor", true);
```

Comment:
I'd make this `backward_blind`, and in all appropriate places to rename / readme

This should be default `false`.

Reply:
That's the mystery of creating names. Thank you!